bliss/api.py

```diff
@@ -317,17 +330,17 @@ def pred_to_astropy_table(pred: Dict[str, Tensor]) -> Table:
 
     bands = ["u", "g", "r", "i", "z"]  # NOTE: SDSS-specific!
     for bnd in bands:
-        pred_table[f"star_log_flux {bnd}_mean"] = pred_table[f"star_log_flux {bnd}_mean"] * log_nmgy
-        pred_table[f"star_log_flux {bnd}_std"] = pred_table[f"star_log_flux {bnd}_std"] * log_nmgy
-        pred_table[f"galsim_flux_{bnd}_mean"] = pred_table[f"galsim_flux_{bnd}_mean"] * u.nmgy
-        pred_table[f"galsim_flux_{bnd}_std"] = pred_table[f"galsim_flux_{bnd}_std"] * u.nmgy
-
-    pred_table["galsim_beta_radians_mean"] = pred_table["galsim_beta_radians_mean"] * u.radian
-    pred_table["galsim_beta_radians_std"] = pred_table["galsim_beta_radians_std"] * u.radian
-    pred_table["galsim_a_d_mean"] = pred_table["galsim_a_d_mean"] * u.arcsec
-    pred_table["galsim_a_d_std"] = pred_table["galsim_a_d_std"] * u.arcsec
-    pred_table["galsim_a_b_mean"] = pred_table["galsim_a_b_mean"] * u.arcsec
-    pred_table["galsim_a_b_std"] = pred_table["galsim_a_b_std"] * u.arcsec
+        pred_table[f"star_log_flux {bnd}_mean"].unit = log_nmgy
+        pred_table[f"star_log_flux {bnd}_std"].unit = log_nmgy
+        pred_table[f"galsim_flux_{bnd}_mean"].unit = u.nmgy
+        pred_table[f"galsim_flux_{bnd}_std"].unit = u.nmgy
```

Comment:
Doesn't need to be done in this PR, but we should probably rename this to `galaxy_flux_*` since the galaxy flux isn't just a galsim parameter

Reply:
Yea, think better addressed in a separate PR, to keep this PR's functionality to mainly handle returning fluxes in nmgy.